rest-api/dao/base_dao.py

```diff
@@ -268,9 +275,12 @@ def _add_pagination_filter(self, query, query_def, fields, first_descending):
     return query.filter(or_(*or_clauses))
 
   def _decode_token(self, query_def, fields):
-    pagination_token = query_def.pagination_token
+    pagination_token = urllib.unquote(query_def.pagination_token.encode("ascii"))
```

Comment:
To avoid breaking existing users, this should ideally continue to accept both old-style and new-style tokens initially, with a TODO to remove old-style token handling in the next release.

Reply:
If an old style token comes in, I can decode it and give them an encrypted token in the reply.  That way nothing breaks and they are moved over to an encrypted token.  Will that work?

Reply:
Yep, exactly